Automattic/woocommerce-payments
# CSV Exports:  Simplify localization by setting export language to that in user profile settings

Fixes #9971


#### Changes proposed in this Pull Request

* Make CSV exports via server localized to the language set in `WP Admin > Users > Profile > Language` 
* Remove redundant code:
  *  Language selection modal that appears when doing CSV export. That looked like this:

![image](https://github.com/user-attachments/assets/b8b4c3a2-d96d-451a-943b-5c95e55431f0)

  * Language selections within `Payments > Settings > Reporting` ( the reporting section will be removed since it has no other options )  Appears like this if you select Spanish as site language: 

<img src="https://github.com/user-attachments/assets/8c9cb919-6493-4a70-9bba-8e63799ed3c9" width="600px" /> 


#### Testing instructions

* Check out develop branch and do these steps:
* Make sure the site has more than 25 transactions or override code to trigger server export only
* Change site language to non-english - Say Spanish or French ( from `WP Admin > Settings > Site language` ) 
* Browse to `Payments > Settings` . Here you should see a section that translates to `Reports`. It should have an option to select the language of the CSV export, similar to the screenshot above. Do not change anything here
* Browse to `Payments > Transactions` section ( menu items will appear in the language you chose ) 
* Try doing a CSV export, it will show a modal similar to the screenshot above. 

* Now checkout this branch
* You should no longer see a `Reports` section within `Payments > Settings`
* Change the site language back to English. 
* Change the language in `User > Profile > Language` to a non English language.
* Clicking export button for CSV, should no longer show a language selection modal
* The exported file should show the correct language as per what was selected within `Users > Profile > Language`
* If you test this on a Pressable site, you should see the email sent in the correct language too. 


#### Notes
* Related issue: #9737 
* Project thread: paJDYF-g5h-p2

-------------------


**Post merge**
